index.html

```diff
@@ -0,0 +1,201 @@
+                        <div>
+                            <img class="pg-icon1" src="images/program_icon_05.png" alt="icons for program-list">
+                        </div>
+                        <p class="pg-info1">Data Analytics with Excel</p>
+                    </div>
+                    <div class="rgt-info">
+                        <p class="rgt-info-txt">In this program, you'll be introduced to the world of data analytics
+                            through
+                            hands-on curriculum developed by Google. You'll develop in-demand data analytics </p>
+                    </div>
+                </div>
+                <div class="program-container">
+                    <div class="lft-info">
+                        <div>
+                            <img class="pg-icon1" src="images/program_icon_04.png" alt="icons for program-list">
+                        </div>
+                        <p class="pg-info1">UX/UI with graphic design</p>
+                    </div>
+                    <div class="rgt-info">
+                        <p class="rgt-info-txt">Get ready for UX Design jobs with hands-on assignments. Learn how to
+                            conduct
+                            user research. Gain the skills, confidence, and experience needed to launch your career in
+                            UX
+                            Design.
+                        </p>
+                    </div>
+                </div>
+            </div>
+
+            <div class="join-us">
+                <button class="action">Join our upcoming cohort</button>
+            </div>
+        </div>
+    </section>
+
+    <!-- featured speakers section -->
+    <section class="tutors">
+        <div class="speaker">
+            <h3>Featured tutors</h3>
+        </div>
+
+        <div class="speaker-container">
+            <div class="speaker-info">
+                <div class="speaker-img">
+                    <img class="thumbnail1_1" src="images/tutor2.png" alt="image of speaker">
+                </div>
+                <div class="bio-container">
+                    <div>
+                        <h4 class="speaker-tag">Lyla Treticov</h4>
+                        <p class="speaker-title">
+                            Secretary General of the Wikimedia Foundation
+                        </p>
+                    </div>
+                    <br>
+                    <div class="speaker-bio">
+                        <p class="name">Layla Treticov is the Executive Director of the Wikimedia Foundation, the
+                            non-profit
+                            organization
+                            that runs Wikipedia. Wikipedia is provided free of charge in 290 languages ​​each month to
+                            over
+                            100 million people, nearly half of the world's population.</p>
+                    </div>
+                </div>
```

Comment:
You did a great job creating the speakers dynamically! Although, keep in mind that all of them should be created with JavaScript, not only 4 😄.